dataactcore/scripts/updateCGAC.py

```diff
@@ -20,21 +21,44 @@ def updateCGAC():
     all_users = userDb.session.query(User).all()
     all_submissions = jobDb.session.query(Submission).all()
 
+    users_to_update = []
+
     for user in all_users:
-        user.cgac_code = findCGACCode(all_cgac, user.cgac_code)
+        if user.cgac_code != None:
+            users_to_update.append(user)
 
-    for submission in all_submissions:
-        submission.cgac_code = findCGACCode(all_cgac, submission.cgac_code)
+    for user in users_to_update:
+        if user.user_id == 2:
```

Comment:
Was this line when you were updating a specific user?  I'm guessing it should be removed if we want this script to update all users?